includes/Story_Post_Type.php

```diff
@@ -54,6 +55,7 @@ class Story_Post_Type extends Service_Base implements Activateable, Deactivateab
 	use Types;
 	use Assets;
 	use Screen;
+	use Rewrite_Flush;
```

Comment:
This is just added here so it can be used in `setup_new_site()`? Not sure I like it :/
Seems a bit fragile.

Perhaps `Rewrite_Flush` makes more sense as a `Rewrites` class or something?

Reply:
A class with a single method that is basically static is a little pointless here. Might just be better off with a function a namespace level. I will take a look.

Reply:
Changed to a function. This is basically an unrelated refactor this PR, so don't feel strongly about it.